coconut/control/control.go

```diff
@@ -504,9 +505,165 @@ func ListWorkflowTemplates(cxt context.Context, rpc *coconut.RpcClient, cmd *cob
 		fmt.Fprintln(o, "no templates found")
 	} else {
 		fmt.Fprintln(o, "available templates in loaded configuration:")
+		var prevRepo string
 		for _, tmpl := range templates {
-			fmt.Fprintln(o, "\t" + tmpl)
+			if (prevRepo != tmpl.GetRepo()) {
+				fmt.Fprintln(o, tmpl.GetRepo())
+			}
+			prevRepo = tmpl.GetRepo()
+			fmt.Fprintln(o, "\t" + tmpl.GetTemplate())
 		}
 	}
 	return nil
+}
+
+// ListRepos lists all available git repositories that are used for configuration.
+func ListRepos(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Command, args []string, o io.Writer) (err error) {
+	if len(args) != 0 {
+		err = errors.New(fmt.Sprintf("accepts no args, received %d", len(args)))
+		return
```

Comment:
naked returns are frown upon (and source of subtle bugs).

```suggestion
		return err
```